.github/workflows/on-push-lint-charts.yml

```diff
@@ -0,0 +1,57 @@
+name: Lint and Test Charts
+
+on: 
+  push:
+    paths:
+      - 'charts/**' 
+      - '.github/**'            
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0        
+          
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Set up kube-score
+        run: |
+          wget https://github.com/zegl/kube-score/releases/download/v1.10.0/kube-score_1.10.0_linux_amd64 -O kube-score
```

Comment:
I would prefer to move the version into an env variable so we can change it in one place.

Reply:
Done :)